chef.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "mixlib-shellout", ">= 2.0.0.rc.0", "< 3.0"
   s.add_dependency "ohai", "~> 8.0"
 
-  s.add_dependency "ffi-yajl", ">= 1.2", "< 3.0"
```

Comment:
Well the other ones are fine with 1.2.x, when we install chef we should get 2.2 which will satisfy the ohai and chef-zero deps and that should be the only one we wind up with.  I don't think this will result in multiple versions being installed, and generally I don't like bumping lower version numbers until I have to in order to keep the compatibility matrix as wide as possible to prevent conflicts.